tc_driver/Cases.jl

```diff
@@ -1193,6 +1296,8 @@ end
 ##### LES_driven_SCM
 #####
 
+# TODO - what to do about those? Could we read in pressure from LES?
```

Comment:
line 1407 you have the thetali profile from LES that can use in a similar way. 
@costachris can you take a look as well?

Reply:
I think it would be better to just take the pressure profile from LES. It should be hydrostatic. 

@costachris do you know if it's available?